integrations/azure/.port/spec.yaml

```diff
@@ -0,0 +1,15 @@
+version: v0.1.0rc12
+type: azure
+description: azure integration for Port Ocean
+icon: Cookiecutter
+features:
+  - type: exporter
+    section: Azure Providers
```

Comment:
This should match the section in port
Talk to dudi where should it be

Reply:
Changed to Cloud Providers, to align with aws and gcp